library/src/main/java/com/mapzen/android/graphics/MapzenMap.java

```diff
@@ -179,7 +184,7 @@ public float getZoom() {
    * Set map position without animation.
    */
   public void setPosition(LngLat lngLat) {
-    mapController.setPosition(lngLat);
+    setPosition(lngLat, 0);
```

Comment:
I'm not sure that calling `mapController.setPositionEased(lngLat, 0, EaseType.CUBIC)` with a duration of `0` is the same as calling `mapController.setPosition(lngLat)`. This could be the source of the issues we are seeing in Eraser Map with setting the position in route preview and routing mode?